routemaster.rb

```diff
@@ -4,11 +4,17 @@ def self.now
   end
 
   DEFAULTS = {
-    redis_pool_size: 1
+    redis_pool_size: 1,
+    process_type:    'unknown',
   }.freeze
 
   def self.configure(**options)
     @_config = DEFAULTS.merge(options)
+    counters.incr('process', type: config[:process_type], status: 'start')
+  end
+
+  def self.teardown
+    counters.incr('process', type: config[:process_type], status: 'stop').finalize
   end
```

Comment:
Looks like these methods are missing tests; might be nice to have one that checks the counters are incremented/decremented for each?